src/index.css

```diff
@@ -29,6 +29,7 @@ limitations under the License.
   --accent-20: #0dbd8b33;
   --alert: #ff5b55;
   --alert-20: #ff5b5533;
+  --warning: #e8bf37;
```

Comment:
This isn't in our design systems from what I can see, supposedly Alert is to be used for all "content requiring user attention"

Reply:
True - I stole it from one of the e2e badges, although probably better to stick with the colour palette.